packages/libs/react-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,80 @@
+import { Tab } from './Tab';
+import { selectorLine, tabsContainer } from './Tabs.css';
+import { TabsPanels } from './TabsPanels';
+
+import React, { FC, ReactNode, useEffect, useRef, useState } from 'react';
+
+export interface ITabsProps {
+  children?: ReactNode;
+  defaultSelected?: string;
+}
+
+interface ITabsComposition extends FC<ITabsProps> {
+  Tab: typeof Tab;
+  Panels: typeof TabsPanels;
+}
+
+// eslint-disable-next-line react/prop-types
```

Comment:
You should be able to remove this by assigning the type

Reply:
Ahh I see what is happening. Discussed this with @bearni95 that we should not be typing the "root" component as a composition because the type evaluation is not correct. The "container"/"root" component is still just a component, not a composition of components so we don't want to assign it an incorrect type and have to cast or disable errors. We've decided to go with the approach that radix takes which is naming the parent component `Tabs.Root`

Reply:
got it